ui/src/components/AppSider/AppSide.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import ScreenNavigation from '../ScreenNavigation/ScreenNavigation'
+import { AppState } from '../../interfaces/storeSlices'
+import { Layout } from 'antd'
+import logo from '../../assets/icons/logo.svg'
+import logoWide from '../../assets/icons/logo-wide.svg'
+import { $do } from '../../actions/types'
+import styles from './AppSide.module.css'
+import { SessionScreen } from '@tesler-ui/core/interfaces/session'
+
+function AppSide() {
```

Comment:
Inconsistent naming, folder is `AppSider`, file and function name is `AppSide`

Reply:
@Dergash renamed